UpdateCTestConfiguration  from :/home/arigachnyy/workspace/github/nbswork/build/cloud/blockstore/libs/notify/ut/DartConfiguration.tcl
UpdateCTestConfiguration  from :/home/arigachnyy/workspace/github/nbswork/build/cloud/blockstore/libs/notify/ut/DartConfiguration.tcl
Test project /home/arigachnyy/workspace/github/nbswork/build/cloud/blockstore/libs/notify/ut
Constructing a list of tests
Done constructing a list of tests
Updating test list for fixtures
Added 0 tests to meet fixture requirements
Checking test dependency graph...
Checking test dependency graph end
test 1
    Start 1: cloud-blockstore-libs-notify-ut

1: Test command: /home/arigachnyy/workspace/github/nbswork/nbs/cloud/blockstore/libs/notify/ut/run_testpack "--print-before-suite" "--print-before-test" "--fork-tests" "--print-times" "--show-fails"
1: Environment variables: 
1:  source_root=/home/arigachnyy/workspace/github/nbswork/nbs
1:  build_root=/home/arigachnyy/workspace/github/nbswork/build
1:  test_split_factor=15
1:  test_split_type=
1: Test timeout computed to be: 60
1: Working dir: /tmp/tmp.af9CZNJVax
1: 
1: Starting notify mock
1: 
1: Traceback (most recent call last):
1:   File "/home/arigachnyy/workspace/github/nbswork/nbs/cloud/blockstore/tests/recipes/notify-mock/__main__.py", line 7, in <module>
1:     import yatest.common as yatest_common
1:   File "/home/arigachnyy/workspace/github/nbswork/nbs/library/python/testing/yatest_common/yatest/__init__.py", line 3, in <module>
1:     from . import common
1:   File "/home/arigachnyy/workspace/github/nbswork/nbs/library/python/testing/yatest_common/yatest/common/__init__.py", line 1, in <module>
1:     from .benchmark import *  # noqa
1:   File "/home/arigachnyy/workspace/github/nbswork/nbs/library/python/testing/yatest_common/yatest/common/benchmark.py", line 3, in <module>
1:     from . import process
1:   File "/home/arigachnyy/workspace/github/nbswork/nbs/library/python/testing/yatest_common/yatest/common/process.py", line 13, in <module>
1:     import packaging.version
1: ModuleNotFoundError: No module named 'packaging'
1: 
1: Notify mock start failed
1: 
1/1 Test #1: cloud-blockstore-libs-notify-ut ...***Failed    0.12 sec
Working dir: /tmp/tmp.af9CZNJVax

Starting notify mock

Traceback (most recent call last):
  File "/home/arigachnyy/workspace/github/nbswork/nbs/cloud/blockstore/tests/recipes/notify-mock/__main__.py", line 7, in <module>
    import yatest.common as yatest_common
  File "/home/arigachnyy/workspace/github/nbswork/nbs/library/python/testing/yatest_common/yatest/__init__.py", line 3, in <module>
    from . import common
  File "/home/arigachnyy/workspace/github/nbswork/nbs/library/python/testing/yatest_common/yatest/common/__init__.py", line 1, in <module>
    from .benchmark import *  # noqa
  File "/home/arigachnyy/workspace/github/nbswork/nbs/library/python/testing/yatest_common/yatest/common/benchmark.py", line 3, in <module>
    from . import process
  File "/home/arigachnyy/workspace/github/nbswork/nbs/library/python/testing/yatest_common/yatest/common/process.py", line 13, in <module>
    import packaging.version
ModuleNotFoundError: No module named 'packaging'

Notify mock start failed



0% tests passed, 1 tests failed out of 1

Label Time Summary:
SMALL    =   0.24 sec*proc (1 test)

Total Test time (real) =   0.12 sec

The following tests FAILED:
	  1 - cloud-blockstore-libs-notify-ut (Failed)
